Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -314,13 +314,9 @@ def rasterize_image_command():
     file_path = demisto.getFilePath(entry_id).get('path')
     filename = f'{entry_id}.pdf'
```

Comment:
only now I noticed this strange naming. maybe it wold have been better to call it `image_to_pdf` and `pdf_to_image`